MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -281,19 +293,16 @@ protected virtual void CreateDeviceResources()
             featureLevels.Add(FeatureLevel.Level_9_1);
 
             var driverType = GraphicsAdapter.UseReferenceDevice ? DriverType.Reference : DriverType.Hardware;
-
-#if DEBUG
+        
             try 
             {
-#endif
                 // Create the Direct3D device.
                 using (var defaultDevice = new SharpDX.Direct3D11.Device(driverType, creationFlags, featureLevels.ToArray()))
                     _d3dDevice = defaultDevice.QueryInterface<SharpDX.Direct3D11.Device1>();
 
                 // Necessary to enable video playback
                 var multithread = _d3dDevice.QueryInterface<SharpDX.Direct3D.DeviceMultithread>();
```

Comment:
Does anyone know why this code is inside the try block?  I didn't add it, but it seems like it should be outside the try/catch in case the device creation fails the first time.